ltm.go

```diff
@@ -977,9 +983,18 @@ func (b *BigIP) CreateMonitor(name, parent string, interval, timeout int, send,
 		ReceiveString: receive,
 	}
 
+	config.cleanMonitor()
+
 	return b.post(config, uriLtm, uriMonitor, parent)
```

Comment:
This method can now call `AddMonitor`. This will help reduce duplication and centralize logic.